src/ncurses.mk

```diff
@@ -50,6 +50,6 @@ define $(PKG)_BUILD
             --with-normal    --without-shared --with-static, \
             --without-normal --without-static --with-shared)
     $(MAKE) -C '$(1)' -j '$(JOBS)'
-    $(MAKE) -C '$(1)' -j 1 install TIC_PATH='$(1).native/progs/tic'
+    $(MAKE) -C '$(1)' -j 1 install -s TIC_PATH='$(1).native/progs/tic'
```

Comment:
I can see the same issue on OSX 10.10 and while both of these allow the build to continue, I suspect they only do so if `tic` is already available.